worksheet_B/Mandelbrot.cpp

```diff
@@ -2,6 +2,125 @@
 //
 
 #include "stdafx.h"
+// source for hsv-rgb "http://stackoverflow.com/questions/3018313/algorithm-to-convert-rgb-to-hsv-and-hsv-to-rgb-in-range-0-255-for-both"
```

Comment:
Quite a lot of code here -- might have been better to split it into a separate file